packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,135 @@
+# Anchor Table Column HLD
+
+## Problem Statement
+
+We need to support links in table cells. We will do this by creating a new column type.
+
+## Links To Relevant Work Items and Reference Material
+
+[GitHub issue: Table allows configuration of hyperlink column type (#1012)](https://github.com/ni/nimble/issues/1012)
+
+## Implementation / Design
+
+This is a fairly straightforward execution of our custom column type pattern. The anchor column will use two fields from the data records: one for the link's visible text (the "label") and one for the url (the "href"). The column configuration will contain the rest of the native anchor parameters: `hreflang`, `ping`, `referrerpolicy`, `rel`, `target`, `type`, and `download`. The assumption is that these should be the same for each link of a given column (if used at all). An alternative would be to have a `*FieldName` property for each of the native anchor parameters, but that seems unlikely to be useful.
+
+If a table record is missing a href value a text span will be rendered rather than a link.
+
+The column will also take an optional `placeholder` value to use when a record does not define a label.
```

Comment:
If my understanding is correct, then the use of the placeholder is more correctly described as follows, right?

```suggestion
The column will also take an optional `placeholder` value to use when a record does not define a label or an href.
```